### PR TITLE
[Debug] ThreeBodyMan

### PR DESCRIPTION
**Pull request by EDGE-qy**
Bug mentioned in #95 fixed referring to #78 
[Skill] SafetyDeclare fit in the Skill Display Board in #78

### CODE REVIEW

File: src/battle_game/core/units/three_body_man.cpp [hunk 1]
```diff
@@ -73,6 +73,9 @@ ThreeBodyMan::ThreeBodyMan(GameCore *game_core, uint32_t id, uint32_t player_id)
           mgr->RegisterModel(turret_vertices, turret_indices);
     }
   }
+  Skill SafetyDeclare("SafetyDeclaration", E, 0, 5 * kTickPerSecond);
+  SafetyDeclare.function = SKILL_ADD_FUNCTION(ThreeBodyMan::SafetyDeclareClick);
```
Comment (LazyJazz) [inline src/battle_game/core/units/three_body_man.cpp:77]: variable name should be in lower_case.

**Review by LazyJazz — CHANGES_REQUESTED** [review]
LGTM! Just fix the nit.

**Comment by EDGE-qy** [comment]
> LGTM! Just fix the nit.

Fixed

**Review by LazyJazz — APPROVED** [review]
LGTM! Call me on wechat if you want to merge this pr into the main branch.

**Pull request closed without merging**